com/TestSources/PyCOMTest/PyCOMTest.idl

```diff
@@ -70,7 +70,7 @@ library PyCOMTestLib
 		const long LongTest2 = 0x7FFFFFFFL;
 		const unsigned char UCharTest = 255;
 		const char CharTest = -1;
-		const LPWSTR StringTest = L"Hello Wo�ld";
+		const LPWSTR StringTest = L"Hello Wo�ld";
```

Comment:
> I don't have a lot of experience with COM but to me it looks like the problem is caused by the definition of `StringTest` as a `LPWSTR`. Skimming a little through my ancient COM literature it looks like this is calling for trouble.

Locally I tried changing this for 
```suggestion
		const WCHAR StringTest[] = L"Hello Wo®ld";
```
(like most other `= L"` that can be found in code) and it seems to pass? (at least it did locally after rebuilding the test and re-registering PyComTest.dll)